provisioning/provisioning-device-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/device/internal/task/Tpm.java

```diff
@@ -0,0 +1,11 @@
+package com.microsoft.azure.sdk.iot.provisioning.device.internal.task;
+
+import com.microsoft.azure.sdk.iot.provisioning.device.TpmRegistrationResult;
+
+public class Tpm extends TpmRegistrationResult
```

Comment:
```suggestion
public class TpmRegistrationResultInternal extends TpmRegistrationResult
```

Maybe we should add a prefix or suffix that could be used for the entire set of such internal setter-only classes?

Reply:
I like this idea, would help make these setters a little less confusing